### PR TITLE
Add support for setting column position of spinner

### LINKED ISSUE #23: Add `indent` option
It would be nice to be able to control what is set to cursorTo in order to support indentation of rows, if desired.

### PR DESCRIPTION
**Pull request by brentonhouse**
- Added `this.column`
- Allows support for indentation w/ spinner

---

Fixes #23

### CODE REVIEW

**Comment by sindresorhus** [comment]
I think `indent` would be a better name for it.

File: index.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ class Ora {
 		this.stream = this.options.stream;
 		this.id = null;
 		this.frameIndex = 0;
+		this.column = 0;
```
Comment (MilosMosovsky) [inline index.js]: `this.column = this.options.column || 0;`

**Comment by brentonhouse** [comment]
- renamed `column` to `indent`
- added suggestion from @MilosMosovsky

File: index.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Ora {
 		this.stream = this.options.stream;
 		this.id = null;
 		this.frameIndex = 0;
+		this.indent = this.options.indent || 0;
```
Comment (sindresorhus) [inline index.js]: You now added an option, so should be documented as such, but I don't really see the use-case of starting with an indented spinner. Do you?

**Comment by sindresorhus** [comment]
Needs a test.

File: index.js (hunk 2 above)
Comment (sindresorhus) [inline index.js]: What do you mean by "embedded"?

Comment (MilosMosovsky) [inline index.js]: @sindresorhus I faced this issue today, i have nested prints in terminal and spinner is starting on very first column which is breaking the overall layout of printed information. It is cosmetic change, but at least 2 persons ran into this issue 😄

Comment (sindresorhus) [inline index.js]: Ok, sounds good, but should be documented as an option too.

**Comment by sindresorhus** [comment]
ping

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Comment by SamVerschueren** [comment]
I believe the initial use case can now be solved with Listr as well.

**Pull request closed without merging**